sample-app/shared/src/wasmJsMain/kotlin/org/publicvalue/multiplatform/oidc/sample/home/HandleRedirect.wasmJs.kt

```diff
@@ -0,0 +1,21 @@
+package org.publicvalue.multiplatform.oidc.sample.home
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import io.ktor.http.*
+import kotlinx.browser.window
+
+@Composable
+actual fun HandleRedirect(state: HomeUiState) {
+    LaunchedEffect(Unit) {
+        val url = Url(window.location.toString())
+
+        val parameters = url.parameters
+
+        val stateParam = parameters["state"]
+        val codeParam = parameters["code"]
+        if(stateParam != null && codeParam != null) {
+            state.eventSink(HomeUiEvent.Redirect(state = stateParam, code = codeParam))
+        }
+    }
```

Comment:
It would be nice to have this code in the appsupport module to make it usable for other apps